packages/calcite-components/src/utils/focusTrapComponent.ts

```diff
@@ -10,6 +10,13 @@ export interface FocusTrapComponent {
   /** When `true`, prevents focus trapping. */
   focusTrapDisabled?: boolean;
 
+  /**
+   * When defined, provides a condition for focusTrapDisabled override.
+   *
+   * @internal
```

Comment:
This doesn't need the `@internal` tag. Should follow similar to `onOpen` from `openCloseComponent.ts`